source/reference/bson-types.txt

```diff
@@ -37,8 +37,8 @@ ObjectId
 --------
 
 ObjectIds are small, likely unique, fast to generate, and ordered.
-ObjectId values consists of 12-bytes, where the first four bytes are a
-timestamp that reflect the ObjectId's creation, specifically:
+ObjectId values consist of 12-bytes, where the first four bytes are a
```

Comment:
You are correct sir!  12 bytes.  Made an update after merging your pull.  Thank you!